Meta/CMake/processor-count.cmake

```diff
@@ -1,3 +1,6 @@
 include(ProcessorCount)
 ProcessorCount(N)
-message("${N}")
+# Executing echo here allows us to print to
+# the standard output, to separate the
+# processor count from potential errors
```

Comment:
This looks like a _really_ short line wrap. You can make those lines almost double as long.